src/components/explore/GenerationResults.tsx

```diff
@@ -18,14 +18,14 @@ const GenerationResults: React.FC<any> = observer((props) => {
   const [solutionModalOpen, setSolutionModalOpen] = React.useState(false);
   const [modalSolution, setModalSolution] = React.useState(workflowSolutions[0]);
 
-  const handleSelected = (solution: WorkflowSolution, checked: boolean) => {
+  const handleSelected = (workflow: WorkflowSolution, checked: boolean) => {
```

Comment:
Not sure about the naming; is it really refering to the workflow? It might be good to emphasize the solution aspect. But I guess solution only doesn't cover the load. Maybe we want to consider workflowSolution for clarity, but that's a bit long...

Reply:
Unless you consider a generated workflow solution to be a workflow, but the terminology might be confusing because we also have the workflow configuration, and the cwl file that also represents a workflow...

Reply:
The initial idea was to use the same naming convention in all the files. we named it `workflow` in 
in src/components/benchmark/VisualizeBenchmarks.tsx 

The solution/result of the synthesis is a workflow after all. 
I see that we have a structure `WorkflowConfig`, maybe that should be renamed into `SynthesisConfig`, considering that it specifies the configuration used to run the synthesis (the solutions to the synthesis problem are workflows).

Reply:
Fair enough, let's rename other stuff then as well, I guess the most important is to be consistent. But that could be another task; I will merge this one